content/en/tracing/trace_collection/custom_instrumentation/ruby/dd-api.md

```diff
@@ -26,6 +26,20 @@ If you have not yet read the instructions for auto-instrumentation and setup, re
 
 This page details describes use cases for adding and customizing observability with Datadog APM.
 
+## Requirements
+
+Make sure you require the appropriate gem for your [Ruby tracer version][8]:
+
+- For v1.x, require the `ddtrace` gem:
+  ```ruby
+  require 'ddtrace'
+  ```
+
+- For v2.x, require the `datadog` gem:
+  ```ruby
+  require 'datadog'
+  ```
```

Comment:
````suggestion
- For v2.x, require the `datadog` gem:
  ```ruby
  require 'datadog'
  ```

- For v1.x, require the `ddtrace` gem:
  ```ruby
  require 'ddtrace'
  ```
````

Let's suggest the latest version first.